app/src/main/res/layout-land/admin_pin_activity.xml

```diff
@@ -10,89 +10,103 @@
       type="org.oppia.app.profile.AdminPinViewModel" />
   </data>
 
-  <ScrollView
-    android:id="@+id/scrollViewAdminPin"
+  <FrameLayout
     android:layout_width="match_parent"
-    android:layout_height="match_parent"
-    android:background="@color/addProfileBackground">
+    android:layout_height="match_parent">
 
-    <androidx.constraintlayout.widget.ConstraintLayout
-      android:id="@+id/admin_auth_container"
+    <ScrollView
+      android:id="@+id/scrollViewAdminPin"
       android:layout_width="match_parent"
-      android:layout_height="wrap_content"
+      android:layout_height="match_parent"
       android:background="@color/addProfileBackground"
-      android:paddingBottom="40dp">
+      android:overScrollMode="never"
+      android:scrollbars="none">
 
-      <TextView
-        android:id="@+id/admin_pin_main_text"
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:layout_marginStart="50dp"
-        android:layout_marginTop="32dp"
-        android:layout_marginEnd="50dp"
-        android:paddingStart="32dp"
-        android:paddingEnd="32dp"
-        android:text="@string/admin_pin_requirement_description"
-        android:textColor="@color/oppiaPrimaryText"
-        android:textSize="16sp"
-        app:layout_constraintEnd_toEndOf="parent"
-        app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toTopOf="parent" />
-
-      <TextView
-        android:id="@+id/admin_pin_warning_text"
+      <androidx.constraintlayout.widget.ConstraintLayout
+        android:id="@+id/admin_auth_container"
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:layout_marginTop="16dp"
-        android:paddingStart="32dp"
-        android:paddingEnd="32dp"
-        android:text="@string/admin_pin_pin_description"
-        android:textColor="@color/oppiaPrimaryText"
-        android:textSize="16sp"
-        app:layout_constraintEnd_toEndOf="parent"
-        app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toBottomOf="@+id/admin_pin_main_text" />
+        android:background="@color/addProfileBackground"
```

Comment:
Can we have background specific to this file only or we can use generalised one to avoid future changes.

Reply:
Updated to `oppiaBackgroundYellowIvory`